### PR TITLE
Check for JIRA ID or "no jira" marker in commit message.

### PR DESCRIPTION
**Pull request by siong-chin**

### CODE REVIEW

File: main/githooks.py [hunk 1]
```diff
@@ -773,6 +781,24 @@ def check_commit_msg(message, files):
...
+            self.assertEqual(bool(m), is_jira)
+        _test('BLD-5704')
+        _test("BLD-1234 fixed some builds")
+        _test("fixed some builds BLD-1234 ")
+        _test("fixed some builds (Jira BLD-1234)")
+        _test("fixed some builds\n some more text BLD-1234")
+        _test('lower-1234', False)
+        _test('A-1234', False)
+        _test('ABCDEF-1234', False)
+        _test('BLD-12345', False)
```
Comment (MatthewT123) [inline main/githooks.py]: I suggest the upper limit on number of issues would be better as six digits - mercury is already at 6000 issues.

Comment (MatthewT123) [reply main/githooks.py]: Also, one project at least has a very long initial ID, even though this was a mistake we need to support it.  I will also suggest adding a positive test for "SKETCHER-1".

File: main.py [hunk 2]
```diff
@@ -14,6 +15,9 @@
 
 if __name__ == '__main__':
 
+    message = os.getenv('INPUT_COMMITMESSAGE')
+    print(f'Commit message: {message}')
```
Comment (MatthewT123) [inline main.py]: I wondered if this should be part of the message below?  Or after it?  This is quite possibly too trivial a comment :)

File: .github/workflows/status_check.yml [hunk 3]
```diff
@@ -14,4 +14,9 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: "3.7"
-      - uses: ccdc-opensource/commit-hooks@v2
+      - name: Get the commit message
```
Comment (MatthewT123) [inline .github/workflows/status_check.yml:17]: I wondered if this was the same message as used for the file size marker?  My expectation was that the same commands would be used for both, but I realise I may be missing something.

**Review by MatthewT123 — CHANGES_REQUESTED** [review]

File: main/githooks.py (hunk 1 above)
Comment (siong-chin) [reply main/githooks.py]: We will update the pattern to [A-Z]{2,8}-[0-9]{1,5}

File: main.py (hunk 2 above)
Comment (siong-chin) [reply main.py]: "After it" feels more natural.

File: .github/workflows/status_check.yml (hunk 3 above)
Comment (siong-chin) [reply .github/workflows/status_check.yml:17]: Yes it's the same message. The action didn't check the commit message (for file size marker, and, for that matter, file size), and I just realised it when doing the JIRA ID.

**Review by MatthewT123 — APPROVED** [review]
These look good to me.  Ironically the files here are failing the pre commit checks, which is probably unavoidable - unless you change the code to concatenate two strings to make up the DO NOT MERGE text, so that the code here can itself pass the check .

**Pull request merged**